.github/workflows/docker.yaml

```
@@ -0,0 +1,65 @@
+name: Docker
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag to use (defaults to "test")'
+        default: "test"
+
+permissions:
+  id-token: write
+  contents: read
+
+env:
+  VERSION: 2.0.0
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
+    env:
+      DOCKER_BUILDKIT: 1
+      BUILDKIT_PROGRESS: plain
+      CLOUDSDK_CORE_DISABLE_PROMPTS: 1
+      IMAGE_NAME: clinvarbitration
+      DOCKER_DEV: australia-southeast1-docker.pkg.dev/cpg-common/images-tmp
```

```suggestion
      DOCKER_DEV: australia-southeast1-docker.pkg.dev/cpg-common/images-dev
```
If you're interested in the images persisting, -dev may be a more suitable option. Noting that tmp is cleaned up regularly.

---

Aye, I'm keen to step over to the `-tmp` artifactory more generally. My mental model is that a `-dev` image should be used for testing a feature, then never again once it's in `main`, so a temporary image seems like a good fit.

Given that moving to a fully containerised approach might mean building images to test features, I don't want to inflate our storage by keeping all the development intermediates.

---

I'll update the attribute name from `DOCKER_DEV` to `DOCKER_TMP` though